app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
As discussed through another channel. This will need to be terminated another time, maybe at the end once we have all the string that we want to translate.

Reply:
Yes that's right, I've actually just translated a few words to show that the system works. In the end, we'll just have to agree on the different translations.